src/operator/numpy/np_elemwise_broadcast_op.h

```diff
@@ -51,6 +51,11 @@ inline void PrintErrorMessage(const std::string& op_name, const int dtype1, cons
              << " yet...";
 }
 
+inline bool is_integer(const int dtype) {
```

Comment:
no need for this extra function, you could use `common::is_float(const int dtype)` instead.